packages/components/src/MultiSelect/MultiSelect.container.test.js

```diff
@@ -1,11 +0,0 @@
-import React from 'react';
-import { shallow } from 'enzyme';
-
-import Container from './MultiSelect.container';
-
-describe('Container MultiSelect', () => {
-	it('should render MultiSelect with props', () => {
-		const wrapper = shallow(<Container />);
-		expect(wrapper.props()).toMatchSnapshot();
-	});
-});
```

Comment:
I removed this test because it doesn't really test anything. It's a render by default, and we print the props, but the props doesn't show anything.
Adding real test is out of scope, especially for this component that we discourage to use.